featurebyte/schema/table.py

```diff
@@ -33,6 +33,35 @@ class TableCreate(FeatureByteBaseModel):
     # pydantic validators
     _columns_info_validator = validator("columns_info", allow_reuse=True)(columns_info_validator)
 
+    @classmethod
+    def _special_column_validator(cls, column_name: str, values: Dict[str, Any]) -> str:
```

Comment:
I saw for other reusable validators that we currently have, they only accept the field under validation in the function signature. Does pydantic automatically inject `values` dict when the signature declares it?

Reply:
Yes it does if defined in the signature